runtime/src/lib.rs

```diff
@@ -755,26 +756,11 @@ impl migration::Trait for Runtime {
     type Event = Event;
 }
 
-// Forum bureaucracy
-impl bureaucracy::Trait<bureaucracy::Instance1> for Runtime {
+// Storage working group
+impl working_group::Trait<working_group::Instance2> for Runtime {
```

Comment:
Why the change to Instance2 ? Are you reserving Instance1 for the content working group? Or was instance1 for the forum.. I kind of got confused here. Maybe we can import working_group::Instance2 and working_group::Instance1 with an alias at the top to avoid this confusion and possibly making a mistake references the wrong instance.

Reply:
Introduced the alias for instances. Instance1 is reserved for the Forum.